deploy/docker-swarm/clickhouse-setup/docker-compose.yaml

```diff
@@ -98,11 +101,8 @@ services:
       # - "14268:14268"   # Jaeger thrift HTTP
       # - "55678:55678"   # OpenCensus receiver
       # - "55679:55679"   # zPages extension
-    environment:
-      - OTEL_RESOURCE_ATTRIBUTES=host.name={{.Node.Hostname}},os.type={{.Node.Platform.OS}},dockerswarm.service.name={{.Service.Name}},dockerswarm.task.name={{.Task.Name}}
     deploy:
-      mode: replicated
-      replicas: 3
+      mode: global
```

Comment:
@prashant-shahi will this break existing swarm users?

Reply:
it will not break, but they won't have 3 replicated contained but rather same number of containers as that of count of Swarm cluster nodes.